HLTrigger/special/plugins/HLTMuonPointingFilter.cc

```diff
@@ -39,6 +39,9 @@ HLTMuonPointingFilter::HLTMuonPointingFilter(const edm::ParameterSet& pset) :
   thePropagatorName(pset.getParameter<std::string>("PropagatorName") ),
   theRadius(        pset.getParameter<double>("radius") ),              // cyl's radius (cm)
   theMaxZ(          pset.getParameter<double>("maxZ") ),                // cyl's half lenght (cm)
+  thenPixHits(      pset.getParameter<int>("nPixHits") ),               // pixel hits
```

Comment:
Can you make the new parameters `unsigned` ?
I assume it makes no sense to ask for `> -1` hits, since `0` means no requirements, right ?